src/geometry/closest_points.rs

```diff
@@ -42,12 +42,12 @@ fn closest_points_aux(
 
     if n <= 3 {
         // bruteforce
-        let mut min = dist(&points[0], &points[1]);
+        let mut min = sqr_dist(&points[0], &points[1]);
         let mut pair = (points[0], points[1]);
 
-        for i in 0..n {
+        for i in 1..n {
```

Comment:
Why change indexing?

Reply:
Because we already computed values for `0` + `1` indexes in a few lines above
```rust
        let mut min = sqr_dist(&points[0], &points[1]);
        let mut pair = (points[0], points[1]);
```

What is the point to recalculate it again for this pair of indexes?